src/main/java/com/esri/samples/raster/raster_rendering_rule/README.md

```diff
@@ -0,0 +1,39 @@
+<h1>Raster Rendering Rule</h1>
+
+<p>Display a raster on a map and apply different rendering rules to that raster.</p>
+
+<p><img src="RasterRenderingRule.png"/></p>
+
+<h2>Use case</h1>
+
+<p>Image Service Rasters may provide several rendering rules options. When it is necessary to switch between rendering rules in order to display the desired data, the interface for the selection and application of the required rule can be used.</p>
+
+<h2>How to use the sample</h2>
+
+<p>Run the sample and use the drop-down menu at the top to select a rendering rule.</p>
+
+<h2>How it works</h2>
+
+<ol>
```

Comment:
This section is meant to be a high-level overview of the steps required to use the API. It should avoid specifics about how its being shown in the sample (anything about the UI). Here's what I'd suggest:

1. Create an `ImageServiceRaster` using a URL to an online image service.
2. After loading the raster, use `imageServiceRaster.getServiceInfo().getRenderingRuleInfos()` to get a list of `RenderingRuleInfo`s supported by the service.
3. Choose a rendering rule info to apply and use it to create a `RenderingRule`.
4. Create a new `ImageServiceRaster` using the same URL.
5. Apply the rendering rule to the new raster using `imageServiceRaster.setRenderingRule(renderingRuleInfo)`
6. Create a `RasterLayer` from the raster for display.

Since this sample does not have a common design readme, please open a PR with the markdown version of this so it can be reviewed by the team.

Reply:
Thanks, I've been working on a few readmes recently and slowly getting the feel for it. The more I work with the API the easier it gets to describe the steps. I'm seeing now the reference to the ComboBox is not ideal.